target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata

```diff
@@ -31,7 +31,9 @@ case "$FIRMWARE" in
 			;;
 		esac
 		;;
-	ubnt,unifi-6-plus)
+	ubnt,unifi-6-plus|\
```

Comment:
Please keep alphabetical order in all scripts.

Reply:
Right, I did not notice it was alphabetically ordered. I moved a few blocks, it should be okay now